README.md

```diff
@@ -0,0 +1,38 @@
+# Simple Node.js Demo
+
+This repository contains a simple Node.js application for demonstration purposes.
+
+## Dependency
+
+- [Node.js](https://nodejs.org/)
+
+## How to Run
+
+1. Clone the repository to your local machine:
+
+    ```bash
+    git clone https://github.com/your-username/simple-node-demo.git
+    ```
+
+2. Navigate to the project directory:
+
+    ```bash
+    cd newton-school-lessons
+    ```
+
+3. Run the application using Node.js:
```

Comment:
To run the application you have to do things:

1. move to the product dir (cd <project name>)
2. install the dependency (npm install)

Can you add instructions for these as well?

Reply:
Yes, will do sir!

Reply:
Hi! Does moving to product dir means going to the specific folder and not just the folder of the repo then running npm install there?

Reply:
Yes. Also I meant project dir instead of product dir, have corrected the last comment.